test/e2e/utils.go

```diff
@@ -391,6 +392,12 @@ func getProviders() ([]TestProvider, error) {
 						},
 					},
 				}
+				fakegcsenabled := getEnvOrFallback(common.EnvGCSEmulatorEnabled, "")
```

Comment:
@anveshreddy18 Could you explain the purpose of expecting the `GOOGLE_EMULATOR_ENABLED` environment variable?

Reply:
I felt checking `GOOGLE_EMULATOR_ENABLED` variable to see if it's enabled is more intuitive than checking the `HOST URL` so I added that. But now to be consistent with other emulators, I've modified it. PTAL